### DISCUSSION TITLE
How significant is the 5 second rule for reading a plot?

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#7380] [Q&A] [answered]
Seriously, it seems very arbitrary.  Why 5 seconds, why not 1 minute, or 10 minute?  Where's the real programmed limit?  To what degree is this advice and to what degree is it significant or required?  I had a raspberry pi that was sometimes hitting 6-7 seconds, but that would mean I need a more powerful computer causing it to not be such an efficient coin. What is the real answer?

**Comment by emlowe** [comment, marked as answer]
Chia consensus requires the block to be created and on the network within 30 seconds (including any network latency) - of which the plot lookup is only a part, so if you are taking longer than 5 seconds for plot lookups, it gets more unlikely you will successfully farm a block and get your rewards. 
Lots of people are successfully farming on the pi

**Reply by number435398** [reply]
So maybe the delays I was experiencing weren't significant enough to cause a problem on my raspberry pi.  Then again, raspberry pis seem only able to connect to around 20 or so drives at once, at best, anyhow.

**Reply by rmica** [reply]
20 drives is 0.2 PB or even more. It's no small thing for one pi.
Also things to look for are: copying a plot to a harvesting drive can cause temporary peaks. On a full stable harvester you could increase the plot_loading_frequency_seconds to a very high delay, and eventually restart the harvester by hand after a storage change.

**Reply by reb0rn21** [reply]
~20s here, all works fine.....
